webapp/channels/src/components/profile_popover/profile_popover_custom_attributes.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {getCustomProfileAttributeFields} from 'mattermost-redux/actions/general';
+import {Client4} from 'mattermost-redux/client';
+import {getCustomProfileAttributes} from 'mattermost-redux/selectors/entities/general';
+
+import type {GlobalState} from 'types/store';
+
+type Props = {
+    userID: string;
+}
+const ProfilePopoverCustomAttributes = ({
+    userID,
+}: Props) => {
+    const dispatch = useDispatch();
+    const [customAttributeValues, setCustomAttributeValues] = useState<Record<string, string>>({});
+    const customProfileAttributeFields = useSelector((state: GlobalState) => getCustomProfileAttributes(state));
+
+    useEffect(() => {
+        const fetchValues = async () => {
```

Comment:
Should we use the `getCustomProfileAttributeValues` from `webapp/channels/src/packages/mattermost-redux/src/actions/general.ts`?

Reply:
We'll create followup tickets for these ones